spec/requests/api/v1/docs/users_spec.rb

```diff
@@ -0,0 +1,139 @@
+require "rails_helper"
+require "swagger_helper"
+
+# rubocop:disable RSpec/EmptyExampleGroup
+# rubocop:disable RSpec/VariableName
+
+RSpec.describe "Api::V1::Docs::Users", type: :request do
+  let(:Accept) { "application/vnd.forem.api-v1+json" }
+  let(:api_secret) { create(:api_secret) }
+  let(:user) { api_secret.user }
+
+  let(:banned_user) { create(:user) }
+  let(:article) { create(:article, user: banned_user, published: true) }
+  let(:comment) { create(:comment, user: banned_user, article: article) }
+
+  before do
+    allow(FeatureFlag).to receive(:enabled?).with(:api_v1).and_return(true)
+    user.add_role(:admin)
+  end
+
+  describe "PUT /users/:id/unpublish" do
+    before do
+      user.add_role(:admin)
+    end
+
+    path "/api/users/{id}/unpublish" do
+      put "Unpublish an User's Articles and Comments" do
```

Comment:
```suggestion
      put "Unpublish a User's Articles and Comments" do
```

(I think?)

Reply:
Google seems pretty split: I see a lot of arguments that it's 'a' because 'user' starts with a consonant sound (y), not a vowel sound.

Some other examples that lead me to using a instead of an:
Did you go to a/an university (I'd say a)
Is electricity a/an utility? (I'd say a)
Is healthcare a/an universal right? (I'd say a)